assets/utils/videoFileHelper.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+VALID_VIDEO_FORMATS = ["WEBM", "MPG", "MP2", "MPEG", "MPE", "MPV", "OGG", "MP4", "M4P", "M4V", "AVI", "WMV", "MOV", "QT", "FLV", "SWF", "AVCHD"]
```

Comment:
would rather raise an error when the video is loaded tbh, I don't think we care too much about the file ext as opposed to 'does the file actually work'. Would also cover the edge case where a video file is corrupted in some way

Reply:
Ah this is a good point. I think in general, we need to add more errors/error handling so that the user can have an idea on what to fix.

Reply:
Since this would break if a private video is in a playlist, I think that it would be best to leave it how it is for now and add the error when we add error handling to the upstream functions.